Gemfile.lock

```diff
@@ -343,5 +343,8 @@ DEPENDENCIES
   will_paginate (>= 3.0.6)
   will_paginate-bootstrap (>= 1.0.1)
 
+RUBY VERSION
```

Comment:
Hi, welcome back! Great to see your work here. 

I'm curious, do you have Gemfile.lock in your .gitignore file? I was wondering why its been added to your changes; just checking! We don't need it to be, I believe. 

I'll go through more of this soon, thank you!